definitions/types/ip-address.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "string",
+  "title": "IP Address",
+  "description": "IPV6 formatted IP Address",
+  "examples": [
+    "0.0.0.0",
+    "8.8.8.8",
+    "127.0.0.1"
+  ],
+  "format": "ipv4"
```

Comment:
I tried added a `message` field in the online validator like we see in the other PR and wasn't able to get the validator to return something different. So for now, we get the built-in message provided by the format.